packages/lodestar/src/network/peers/peerManager.ts

```diff
@@ -0,0 +1,443 @@
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {phase0} from "@chainsafe/lodestar-types";
+import {ILogger, LodestarError} from "@chainsafe/lodestar-utils";
+import PeerId from "peer-id";
+import {IBeaconChain} from "../../chain";
+import {GoodByeReasonCode, GOODBYE_KNOWN_CODES, Libp2pEvent, Method} from "../../constants";
+import {IBeaconMetrics} from "../../metrics";
+import {NetworkEvent, INetworkEventBus} from "../events";
+import {IReqResp} from "../reqresp";
+import {Libp2pPeerMetadataStore} from "./metastore";
+import {PeerDiscovery} from "./discover";
+import {IPeerRpcScoreStore, ScoreState} from "./score";
+import {
+  getConnectedPeerIds,
+  PeerMapDelay,
+  SubnetMap,
+  RequestedSubnet,
+  assertPeerRelevance,
+  prioritizePeers,
+} from "./utils";
+
+export interface IPeerManager {
```

Comment:
Indeed the `IPeerManager` is useless. It can be a private property on Network. However `Network` is used everywhere, and referencing the interface instead of the actual class maintains allows for some flexibility and is consistent with other code. We can discuss in an issue our pattern of interfaces.